content/en/docs/usage/gateway-api.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "Securing Gateway Resources"
+linkTitle: "Securing Gateway Resources"
+weight: 90
```

Comment:
Should we change the weight, so that it sits next to the other ingress page in the list?

Reply:
Thanks! I changed the weight to 105 and now it shows right after the "Securing Ingress resources":


<img alt="" src="https://user-images.githubusercontent.com/2195781/127701434-d5bb648f-c0f4-48f7-b24d-b5be819ce0e2.png" width="300">